src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1139,10 +1148,23 @@ private function addSerializerSection(ArrayNodeDefinition $rootNode, callable $e
         $rootNode
             ->children()
                 ->arrayNode('serializer')
+                    ->validate()
+                    ->always(function ($v) {
+                        if (isset($v['enable_annotations'])) {
+                            trigger_deprecation('symfony/framework-bundle', '6.4', 'Option "enable_annotations" at "framework.serializer" is deprecated. Use the "enable_attributes" option instead.');
+
+                            if (!isset($v['enable_attributes'])) {
+                                $v['enable_attributes'] = $v['enable_annotations'];
+                            }
```

Comment:
Should we throw if both settings are set, especially if they don't match?

Reply:
Updated to throw if both options are set :+1: